packages/commonwealth/server.ts

```diff
@@ -31,10 +46,36 @@ const app = express();
  * - Once we fully decouple the models, we can remove the import from `main.ts` that's causing this issue
  */
 const start = async () => {
+  const { models } = await import('@hicommonwealth/model');
+
+  // TODO: evaluate other options for one-off event processing
+  // exit if we have performed a one-off event
+  if (SEND_EMAILS) {
+    const { sendBatchedNotificationEmails } = await import(
+      './server/scripts/emails'
+    );
+    const rc = await sendBatchedNotificationEmails(models);
+    process.exit(rc);
+  }
+
   const { main } = await import('./main');
-  main(app)
+  main(app, models, {
+    port: +PORT,
+    noGlobalActivityCache: process.env.NO_GLOBAL_ACTIVITY_CACHE === 'true',
+    withLoggingMiddleware: true,
+    withStatsMiddleware: true,
+    withFrontendBuild: !NO_CLIENT,
+    withPrerender:
+      PRODUCTION && !NO_PRERENDER && SERVER_URL.includes('commonwealth.im'),
+  })
     .then(() => {
       isServiceHealthy = true;
+      // database clean-up jobs (should be run after the API so, we don't affect start-up time
+      // TODO: evaluate other options for maintenance jobs
```

Comment:
Open to options here - only requirement is that these maintenance jobs need to run locally as well. That's the only reason we have this here instead of just in Heroku scheduler

Reply:
Should we start a new 🪣  to track "maintenance" policies? In terms of project structure, we can place them in a separate package, with their own local scripts, and Heroku procs. The main goal is to decouple maintenance jobs from the main application process.

Reply:
Yea so that's the problem, the main application process is the most frequently executed process in dev env so linking maintenance to it means ICs don't need to run other scripts for maintenance on a regular basis. If we separate the maintenance from the main application process then we should modify the `yarn start` command to include running maintenance (but separate from executing `server.ts`).

Reply:
With time this might become a hidden feature, specially to new developers. I'm for having an explicit "maintenance" workflow, with all policies clearly defined.